docs/Doxyfile

```diff
@@ -770,7 +770,7 @@ WARN_LOGFILE           =
 # spaces.
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = include src/common
+INPUT                  = include src/common cpp-package/include/mxnet-cpp
```

Comment:
Is this combining the C and C++ API together in one Doxygen output?

Reply:
I dont have the dev web server that is publicly accessible for preview.